compose/material/material/src/commonMain/kotlin/androidx/compose/material/Strings.kt

```diff
@@ -31,8 +31,6 @@ internal value class Strings private constructor(@Suppress("unused") private val
         val SliderRangeStart = Strings(5)
         val SliderRangeEnd = Strings(6)
         val SnackbarPaneTitle = Strings(7)
-        // When adding values here, make sure to also add them in material/build.gradle,
-        // updateTranslations task (stringByResourceName parameter), and re-run the task
```

Comment:
Where this info can be found now?

Reply:
Wrote StringsTest with reflection that fails if some value is missed